### PR TITLE
Password: Fixes #1565

### LINKED ISSUE #1565: Password: Hard to trigger
It would great to expand the triggering for this IA. I couldn't figure out how to trigger it until I found the IA page, which a random user probably wouldn't do.

Some queries I tried:

- generate password
- generate strong password
- generate random password
- password generator
- random password generator
- random strong password
- random password 16 characters
- create random password

-----
IA Page: https://duck.co/ia/view/password

### PR DESCRIPTION
**Pull request by cinlloc**
A simple fix, maybe not sufficiently filtering triggers. Let me know what you think about it... 

-----
IA Page: https://duck.co/ia/view/password

### CODE REVIEW

File: lib/DDG/Goodie/Password.pm [hunk 1]
```diff
@@ -19,7 +19,7 @@ topics 'cryptography';
 zci answer_type => 'pw';
 zci is_cached   => 0;
 
-triggers start => 'password', 'random password', 'pw', 'random pw', 'pwgen';
+triggers startend => 'password', 'random password', 'generate password', 'generate random password', 'password generator', 'pw', 'random pw', 'pwgen';
```
Comment (moollaza) [inline lib/DDG/Goodie/Password.pm]: @cinlloc we can keep this to `start` unless we expect these words to appear at the end of a mult-word query (which might not be ideal)

File: lib/DDG/Goodie/Password.pm [hunk 2]
```diff
@@ -49,13 +49,11 @@ my $strengths = join('|', keys %pw_strengths);
 
 handle remainder => sub {
 
-    my $query = shift;
-
-    return if ($query && $query !~ /^(?<fw>\d+|$strengths|)\s*(?<sw>\d+|$strengths|)$/i);
```
Comment (moollaza) [inline lib/DDG/Goodie/Password.pm]: this is still needed to ensure the query matches our expectation. We can't assume that any query that matches our triggers above is a valid query for this IA. For example "password test" now triggers this IA and returns a random password, which isn't good

File: t/Password.t [hunk 3]
```diff
@@ -91,6 +83,87 @@ ddg_goodie_test(
...
+    ),
+    'random password strong 15' => test_zci(
+        qr/.{15} \(random password\)/,
+        structured_answer => {
+            input     => ['15 characters', 'high strength'],
+            operation => 'Random password',
+            result    => qr/^.{15}$/
+        }
+    ),
     'password 65' => undef,
```
Comment (moollaza) [inline t/Password.t]: we should add some more examples that shouldn't trigger such as `password`, `password strength`, etc

Comment (cinlloc) [inline t/Password.t]: Ok for adding non-passing tests, but I think `password` should still trigger IA, shoudn't it?

**Comment by moollaza** [comment]
@cinlloc thanks for updating, apologies for the delay. I've set this up for deployment to Beta and will review it shortly.

We appreciate your patience!

**Comment by moollaza** [comment]
@cinlloc this LGTM :+1:

**Pull request merged**